src/NUnitFramework/framework/Diagnostics/ProgressTraceListener.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+using System.Diagnostics;
+#if !NET6_0_OR_GREATER
+using System.Security.Permissions;
+#endif
+
+namespace NUnit.Framework.Diagnostics
+{
+    /// <summary><para>
+    /// The ProgressTraceListener class allows directing tracing or
+    /// debugging output to <see cref="TestContext.Progress"/>.
+    /// </para><para>
+    /// To activate, place the following snippet into the one-time
+    /// set-up method of either a test's fixture or the set-up
+    /// fixture of a project:
+    /// <c>
+    /// System.Trace.Listeners.Add(new ProgressTraceListener());
```

Comment:
```suggestion
    /// _progressTraceListener = new ProgressTraceListener();
    /// System.Trace.Listeners.Add(_progressTraceListener);
```

This better aligns with the suggested tear-down below.

Reply:
Well, the tear-down is only an alternative proposal:

> Or again remove...

The standard use can remain...

> System.Trace.Listeners.Add(new ProgressTraceListener());

...and let the listener be [closed by the framework](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.trace.close):

> Flushes the output buffer, and then closes the [Listeners].

I'll push another proposal to make the comment clearer.

Reply:
I prefer the user that creates the object is responsible for disposal.